modules/cnv-viewing-node-network-state.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-node-network-state.adoc
+
+[id="cnv-viewing-node-network-state_{context}"]
+= Viewing node network state
+
+A `NodeNetworkState` object exists on every node in the cluster. This object is periodically updated and captures the state of the network for that node. 
+
+.Procedure
+
+. List all the `NodeNetworkState` objects in the cluster:
++
+----
+$ oc get nns
+----
+
+. Inspect a `NodeNetworkState` to view the network on that node. The output in this example has been redacted for clarity:
++
+----
+$ oc get nns node01 -o yaml
+----
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkState
+metadata:
+  generation: 1
+  name: node01 <1>
+  ownerReferences:
+  - apiVersion: v1
+    kind: Node
+    name: node01
+status:
+  currentState: <2>
+    dns-resolver:
+...
+    interfaces:
+    - bridge:
```

Comment:
I would drop the half-empty example of the bridge and just listed `...` under `interfaces`. I find this a little confusing (but don't have any good explanation, it's just my gutfeeling).